src/geometry/quaternion_construction.rs

```diff
@@ -374,7 +374,8 @@ where
     }
 
     /// The unit quaternion needed to make `a` and `b` be collinear and point toward the same
-    /// direction.
+    /// direction. Returns `None` if both `a` and `b points in opposite direction, as then the
```

Comment:
```suggestion
    /// direction. Returns `None` if both `a` and `b` are collinear and point to opposite directions, as then the
```
Using `point to opposite directions` alone may be a bit ambiguous as some people may think this is the case as soon as their dot product is negative.